File: TsogosunProfileAdminWeb/src/app/model/Dtos/profile-admin/patronFreePlayDto.ts
export interface PatronFreePlayDto {
    iD: number;
    vendor?: string | undefined;
    xtraCreditFullDescription?: string | undefined;
    startDate: Date;
    endDate: Date;
    cost: number;
    expiryDate?: Date | undefined;
    redemptionType?: string | undefined;
    mobileImageURL?: string | undefined;
    offerStatus?: string | undefined;
    takeupMedium?: string | undefined;
    dateReceived?: Date | undefined;
    loadedTo?: string | undefined;
    offerAvailability?: string | undefined;
    otpNumber?: number | undefined;
}

File: TsogosunProfileAdmin/MSProfileAdmin/Model/Dtos/PatronFreePlayDto.cs
using System;

namespace tsogosun.com.MSProfileAdmin.Model.Dtos
{
    public class PatronFreePlayDto
    {
        public long ID { get; set; }
        public string Vendor { get; set; }
        public string XtraCreditFullDescription { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public int Cost { get; set; }
        public DateTime? ExpiryDate { get; set; }
        public string RedemptionType { get; set; }
        public string MobileImageURL { get; set; }
        public string OfferStatus { get; set; }
        public string TakeupMedium { get; set; }
        public DateTime? DateReceived { get; set; }
        public string LoadedTo { get; set; }
        public string OfferAvailability { get; set; }
        public long? OtpNumber { get; set; }

    }
}
